mllib/src/main/scala/org/apache/spark/mllib/feature/Word2Vec.scala

```diff
@@ -520,7 +537,7 @@ class Word2VecModel private[spark] (
    * @return array of (word, cosineSimilarity)
    */
   @Since("1.1.0")
-  def findSynonyms(vector: Vector, num: Int): Array[(String, Double)] = {
+  def findSynonyms(vector: Vector, num: Int, norm: Boolean): Array[(String, Double)] = {
```

Comment:
will add java doc to explain it.
for backward compatibility, I can add default value in this function instead of the newly introduced one.